.github/workflows/code_coverage_disable_check.py

```
@@ -0,0 +1,155 @@
+                    ):
+                        file_path = os.path.join(root, file_name)
+                        if has_code_coverage_disable(file_path):
+                            print(f"File {file_path} contains code coverage disable statement.")
+                            code_coverage_found = True
+        elif os.path.isfile(item):
+            # If it's a file, check it directly
+            if (
+                item.endswith(".tsx")
+                or item.endswith(".ts")
+            ) :
+                if has_code_coverage_disable(item):
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)